python/sdk/prelude_sdk/controllers/jobs_controller.py

```diff
@@ -0,0 +1,34 @@
+from prelude_sdk.controllers.http_controller import HttpController
+from prelude_sdk.models.account import verify_credentials
+
+
+class JobsController(HttpController):
```

Comment:
same vein - there's some things you can delete from the DetectController now

Reply:
(think there are some CLI cmds that are using the ones in DetectController, so hunt those down and switch em pls)

Reply:
(the export_scm sdk test also uses DetectController.get_background_job)